sql-plugin/src/main/spark350/scala/com/nvidia/spark/rapids/shims/v2WriteCommandMetasShim.scala

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2025, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*** spark-rapids-shim-json-lines
+{"spark": "350"}
+{"spark": "351"}
+{"spark": "352"}
+{"spark": "353"}
+{"spark": "354"}
+{"spark": "355"}
+{"spark": "356"}
+{"spark": "400"}
```

Comment:
sort by Spark versions.

Reply:
For Iceberg, we only support Spark35x, why add support for 400?
And could we pass the building for other Spark versions 32x, 33x?

Reply:
The reason we need to add 400+ is to add `ReplaceDataExec` for 400, which is not iceberg specific, see https://github.com/liurenjie1024/spark-rapids/blob/d7a13812c6df3e22330bf2d31fa15af4db0247ab/sql-plugin/src/main/spark350/scala/com/nvidia/spark/rapids/shims/Spark350PlusNonDBShims.scala#L180.

In fact, iceberg support will not happen in 400, see https://github.com/liurenjie1024/spark-rapids/blob/7f6739c935410b44d569564fff6c26250fc5b0a5/sql-plugin/src/main/spark350/scala/com/nvidia/spark/rapids/shims/ExternalSourceShim.scala#L42

>And could we pass the building for other Spark versions 32x, 33x?

Yes, our ci will check for build for 32x, 33x.